hilti/runtime/src/tests/bytes.cc

```diff
@@ -84,13 +84,13 @@ TEST_CASE("decode") {
 TEST_CASE("extract") {
     SUBCASE("sufficient data") {
         unsigned char dst1[3] = {0};
-        CHECK_EQ("123456"_b.extract(dst1), "456"_b);
+        CHECK_EQ("123456"_b.extract(dst1, 3), "456"_b);
```

Comment:
Let's get rid of the magic values here and below,

```suggestion
        CHECK_EQ("123456"_b.extract(dst1, sizeof(dst1) / sizeof(dst1[0])), "456"_b);
```